app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -332,6 +332,9 @@ class BrowserFragment :
             owner = this,
             view = rootView
         )
+        if (requireComponents.settings.autoplayState.reloadSession) {
+            requireComponents.sessionUseCases.reload(tabId)
+        }
```

Comment:
Don't see where this `reloadSession` is reset to `false`.
Would it mean that once set to `true` everytime the user navigates back to this Fragment the session would be reloaded?